samples/aspnetcore/blazor/FlightFinder/FlightFinder.Client/Components/SearchResults.razor

```diff
@@ -4,8 +4,8 @@
         <div class="title">
             <h2 class="my-3">@Itineraries.Count results</h2>
             <select class="custom-select" @bind="chosenSortOrder">
-                <option value=@SortOrder.Price>Cheapest</option>
-                <option value=@SortOrder.Duration>Quickest</option>
+                <option value="@SortOrder.Price">Cheapest</option>
+                <option value="@SortOrder.Duration">Quickest</option>
```

Comment:
The quotes aren't required, but previously the sample was inconsistent about where they were applied. I've added them in all the places I spotted they were missing.